music/src/core/player.ts

```diff
@@ -541,3 +541,146 @@ export class PlayerWithClick extends Player {
     super(true, callbackObject);
   }
 }
+
+/**
+ * A `NoteSequence` player that uses a MIDI output for playing. Note that
+ * WebMIDI is not supported in all browsers. A
+ * [polyfill](https://cwilso.github.io/WebMIDIAPIShim/) exists, but it too
+ * requires a plugin to be installed on the user's computer, so it might not
+ * work in all cases.
+ *
+ * If you want to use a particular MIDI output port, you must update the
+ * `output` property before calling `start`, otherwise a message will be sent to
+ * all connected MIDI outputs:
+ *
+ * Example (easy mode):
+ *
+ *   `
```

Comment:
is this the right way to do it or should it be ``` for markdown?

Reply:
great question! i copied it from the SoundfontPlayer docs, but lemme build the docs and see if it's borked